Modules/Devices/src/Devices.Domain/Entities/OAuthClient.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Backbone.Modules.Devices.Domain.Entities;
+public class OAuthClient
+{
+    public OAuthClient(string clientId, string displayName)
+    {
+        ClientId = clientId;
+        DisplayName = displayName;
+    }
+    public string ClientId { get; set; }
+
+    public string DisplayName { get; set; }
```

Comment:
How many times do I have to say that domain class properties should not have public setters?